app/controllers/UploadController.scala

```diff
@@ -23,31 +23,48 @@ class UploadController @Inject()(val controllerComponents: SecurityComponents,
 
   def uploadPage(consignmentId: UUID): Action[AnyContent] = secureAction.async { implicit request: Request[AnyContent] =>
     val consignmentStatusService = new ConsignmentStatusService(graphqlConfiguration)
-    val isJudgmentUser = request.token.isJudgmentUser
 
     for {
       consignmentStatus <- consignmentStatusService.consignmentStatus(consignmentId, request.token.bearerAccessToken)
     } yield {
       val transferAgreementStatus: Option[String] = consignmentStatus.flatMap(_.transferAgreement)
       val uploadStatus: Option[String] = consignmentStatus.flatMap(_.upload)
+      val pageHeading = "Uploading records"
 
       transferAgreementStatus match {
         case Some("Completed") =>
           uploadStatus match {
             case Some("InProgress") =>
-              Ok(views.html.standard.uploadInProgress(consignmentId)).uncache()
+              Ok(views.html.uploadInProgress(consignmentId, pageHeading)).uncache()
             case Some("Completed") =>
-              Ok(views.html.standard.uploadHasCompleted(consignmentId)).uncache()
+              Ok(views.html.uploadHasCompleted(consignmentId, pageHeading)).uncache()
             case _ =>
-              if(isJudgmentUser) {
-                Ok(views.html.judgment.judgmentUpload(consignmentId, frontEndInfoConfiguration.frontEndInfo)).uncache()
-              } else {
-                Ok(views.html.standard.upload(consignmentId, frontEndInfoConfiguration.frontEndInfo)).uncache()
-              }
+              Ok(views.html.standard.upload(consignmentId, frontEndInfoConfiguration.frontEndInfo)).uncache()
           }
         case _ =>
           Redirect(routes.TransferAgreementController.transferAgreement(consignmentId))
       }
     }
   }
+
+  def judgmentUploadPage(consignmentId: UUID): Action[AnyContent] = secureAction.async { implicit request: Request[AnyContent] =>
+    val consignmentStatusService = new ConsignmentStatusService(graphqlConfiguration)
+    val isJudgmentUser = request.token.isJudgmentUser
```

Comment:
Doesn't look like `isJudgmentUser` is called any where, so can be deleted

Reply:
Yes; this was leftover code